omniscidb/Tests/ResultSetArrowConversion.cpp

```diff
@@ -385,6 +385,96 @@ TEST(ArrowTable, EmptySelection) {
   ASSERT_EQ(table->num_rows(), (int64_t)0);
 }
 
+TEST(ArrowTable, LimitedSelection) {
+  auto res =
+      runSqlQuery("select bi from test limit 4 offset 1;", ExecutorDeviceType::CPU, true);
```

Comment:
This and similar queries might not produce the desired `ResultSet`. This is because for such projections it's simple to not include skipped rows in the execution phase. So, you should use sort and/or groupby to produce different ResultSet layouts.

Reply:
Fixed, check please.